CDP4Reporting/CDP4Reporting.csproj

```diff
@@ -19,8 +19,7 @@
      <PackageTags>CDP COMET ECSS-E-TM-10-25</PackageTags>
      <PackageLicenseExpression>LGPL-3.0-only</PackageLicenseExpression>
      <PackageReleaseNotes>
-        [UPGRADE] CDP4Common 10.0.0
-        [Remove] net461 target framework
+         [FIX] OptionDependentDataCollector SelectOption should set selected option]
```

Comment:
can you add rephrase to:

[FIX] OptionDependentDataCollector SelectOption should set selected option; fixes #260